tests/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -pedantic -Werror")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -pedantic -Werror -DBOOST_HANA_CONFIG_ENABLE_STRING_UDL=1")
```

Comment:
Let's do not mix defines and flags in one string literal.

Reply:
Ok